pkg/registries/registry.go

```diff
@@ -181,3 +195,63 @@ func createFilter(config Config, log *logging.Logger) Filter {
 
 	return filter
 }
+
+func validateSpecs(log *logging.Logger, inSpecs []*apb.Spec) []*apb.Spec {
+	var wg sync.WaitGroup
+	wg.Add(len(inSpecs))
```

Comment:
@shawn-hurley I'd like to also talk about this method. It's not in the scope of this PR, but where do we want to actually validate what we actually bring in, and how?